### PR TITLE
DAPI-1461: Update template for Forgotten password pages

### PR DESCRIPTION
**Pull request by olivierpontier**
**Description (for Contributor and Core Developer)**

This pull requests defines a new template for the forgotten password pages.
We are still using Twig templates with Less stylesheets

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Akeneo/UserManagement/Bundle/Controller/ResetController.php [hunk 1]
```diff
@@ -141,14 +141,15 @@ public function checkEmail()
      */
     public function reset($token)
     {
-        $user = $this->userManager->findUserByConfirmationToken($token);
+        //$user = $this->userManager->findUserByConfirmationToken($token);
+        $user = $this->userManager->findUserByUsername('admin');
```
Comment (rouliane) [inline src/Akeneo/UserManagement/Bundle/Controller/ResetController.php]: Modifications to remove ? As far as I know, we're not supposed to edit PHP files for this subject

File: src/Akeneo/UserManagement/Bundle/Controller/ResetController.php [hunk 2]
```diff
@@ -141,14 +141,15 @@ public function checkEmail()
...
 
         if (null === $user) {
             throw $this->createNotFoundException(
                 sprintf('The user with "confirmation token" does not exist for value "%s"', $token)
             );
         }
-
+/*
```
Comment (rouliane) [inline src/Akeneo/UserManagement/Bundle/Controller/ResetController.php]: idem here

File: src/Akeneo/UserManagement/Bundle/Controller/ResetController.php [hunk 3]
```diff
@@ -157,7 +158,7 @@ public function reset($token)
 
             return $this->redirect($this->generateUrl('pim_user_reset_request'));
         }
-
+*/
```
Comment (rouliane) [inline src/Akeneo/UserManagement/Bundle/Controller/ResetController.php]: and here

**Review by rouliane — APPROVED** [review]

File: src/Akeneo/UserManagement/Bundle/Controller/ResetController.php (hunk 1 above)
Comment (olivierpontier) [reply src/Akeneo/UserManagement/Bundle/Controller/ResetController.php]: This is dev stuff for testing reset password form page. I should not have to commit this file. Sorry!

**Review by AymericPlanche — APPROVED** [review]

**Pull request merged**